### PR TITLE
Add "Edit page" button to desktop ToC

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Static layout uses the table of contents for both mobile and desktop (while the others only use for mobile). The desktop version of the ToC will render a "Edit page" button if an `editPath` is provided.

- Provides `editPath` for the desktop usage of the ToC
- Forms the edit path from a new temporary env var `NEXT_PUBLIC_GITHUB_REPO=ethereum/ethereum-org-fork`, `CONTENT_DIR`, `slug` and the string `index.md`
- An env var was used to allow switching between the two repos from the build settings instead of the code. For development, we need to point this call to either the Gatsby repo with `src/content`, or the new repo with `public/content`... Mixing will not work. Since we have `CONTENT_DIR` already pointing to `public/content`, I decided to point to an env var with a fallback of `ethereum/ethereum-org-website`. I added the env var to Netlify with a value of `ethereum/ethereum-org-fork` which can be deleted at time of production.

## Related Issue
https://www.notion.so/efdn/Add-Edit-Button-w-proper-slug-to-TableOfContents-component-724200da64074caf85eb210d7cef2ef6?pvs=4

### CODE REVIEW

File: src/components/TableOfContents/index.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const TableOfContents: React.FC<IProps> = ({
       >
         <List {...outerListProps}>
           {!hideEditButton && editPath && (
-            <ListItem mb={2}>
+            <ListItem mb={4}>
```
Comment (pettinarip) [inline src/components/TableOfContents/index.tsx]: Lets try to minimize these style changes as much as we can in this migration

**Review by pettinarip — APPROVED** [review]
LGTM!

File: src/components/TableOfContents/index.tsx (hunk 1 above)
Comment (wackerow) [reply src/components/TableOfContents/index.tsx]: 😢 lol

Comment (wackerow) [reply src/components/TableOfContents/index.tsx]:
```suggestion
            <ListItem mb={2}>
```

**Pull request merged**